dist/css/material-wfont.css

```diff
@@ -30890,6 +30896,47 @@ icon-material-blue-grey-A700 {
 .panel-footer {
   background-color: #ececec;
 }
+hr.on-dark {
+  color: #1a1a1a;
+}
+hr.on-light {
+  color: #ffffff;
+}
+@media (-webkit-min-device-pixel-ratio: 0.75), (min-resolution: 120dpi) {
```

Comment:
Seems autoprefixer doesn't support pixel density yet, targeted for 5.1 - I'll add in all the vendor prefixes.